### ISSUE TITLE
list-field with no duplicates allowed?

### ISSUE THREAD

**Issue by joemienko** [redgeoff/mson-react#126]
Is it possible to easily prevent the entry of duplicates in a list-field?

For example, a list could contain `{dog, cat, fish}`...

But not `{dog, dog, cat, fish}`.

Thanks, for any help you can provide, and thanks for an awesome library!

**Comment by redgeoff** [comment]
Yeah, this is a nice feature and I've wanted to add it. For now, if you are looking to implement this for just a list of strings then you can use the `SelectField` with the `multiple` property set to `true`.
